python/send_message/main.py

```diff
@@ -0,0 +1,36 @@
+from .functions.send_sms_twilio import send_sms_twilio
+from .functions.send_email_mailgun import send_email_mailgun
+from .functions.send_tweet import send_tweet
+from .functions.send_message_discord_webhook import send_message_discord_webhook
+
+
+def main(req, res):
+    """Main Function"""
+
+    global result
+
+    try:
+        payload = req.payload
+
+        payload_type = payload["type"]
+        message = payload["message"]
+
+        if payload_type == "SMS":
+            reciever = payload["receiver"]
+            result = send_sms_twilio(req.variables, reciever, message)
+        elif payload_type == "Email":
+            reciever = payload["receiver"]
+            subject = payload["subject"]
+            result = send_email_mailgun(req.variables, reciever, message, subject)
+        elif payload_type == "Twitter":
+            result = send_tweet(req.variables, message)
+        elif payload_type == "Discord":
+            result = send_message_discord_webhook(req.variables, message)
+        else:
+            result = {"success": False, "error": "Invalid Type"}
+
+    except Exception as e:
+
+        return res.json({"success": False, "error": e})
```

Comment:
This fails because e is not serializable:

```suggestion
        return res.json({"success": False, "error": str(e)})
```

Please make sure this is addressed in the rest of your code too.